client/src/components/ReviewTiles/index.jsx

```diff
@@ -7,6 +7,12 @@ const ReviewTiles = ({ review }) => {
   const {
     rating, reviewer_name, date, summary, body, helpfulness,
   } = review;
+  const monthOptions = ['January ', 'February ', 'March ', 'April ', 'May ', 'June ', 'July ', 'August ', 'September ', 'October ', 'November ', 'December '];
+  const monthNumber = new Date(date).getMonth();
+  const month = monthOptions[monthNumber];
+  const day = new Date(date).getDate();
+  const year = new Date(date).getFullYear();
+  const formattedDate = `${month} ${day}, ${year}`;
```

Comment:
next time create only one `new Date()` object and invoke getDate, getMonth, etc on it

Reply:
Great catch, Phil! The reasoning behind this was because when I used getMonth alone, I would get a number, but the business requirements wanted a fully spelled out month - which is why I created an array to get the month. I also tried using new Date (const formattedDate = new Date(date)) without the chaining, but for some reason it only works when console.logged but would crash the app when I tried to implement - so this was my quick fix. Can revisit and refactor later as future enhancement.